CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set_target_properties(crypto3_precompiled_headers PROPERTIES
     CXX_STANDARD_REQUIRED TRUE
 )
 
+if (ENABLE_TESTS)
+    enable_testing()
+endif()
+
```

Comment:
Please, wait for the CI of this PR
https://github.com/NilFoundation/placeholder/pull/213

Let's see, if it changes the number of tests

Reply:
As I see, it's exactly about this line...
https://github.com/NilFoundation/placeholder/pull/213